src/main/java/wolox/training/controllers/GreetingController.java

```diff
@@ -0,0 +1,18 @@
+package wolox.training.controllers;
+
+import org.jetbrains.annotations.NotNull;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class GreetingController {
+
+    @GetMapping("/greeting")
+    public String greeting(@RequestParam(name="name", required=false, defaultValue="World") String name, @NotNull Model model) {
```

Comment:
In this case, you are using 'name' and 'greeting' more than one time.

It's always a good practice to extract this strings on a constant, to reuse it.